authorizerd.go

```diff
@@ -89,21 +91,22 @@ type authorizer struct {
 	jwkErrRetryInterval string
 
 	// accessTokenProcessor parameters
-	atpParams []ATProcessorParam
+	accessTokenParam AccessTokenParam
 
 	// roleTokenProcessor parameters
-	verifyRoleToken bool
+	enableRoleToken bool
 	rtHeader        string
 
 	// roleCertificateProcessor parameters
-	verifyRoleCert bool
+	enableRoleCert bool
 }
 
 type mode uint8
 
 const (
 	token mode = iota
 	jwt
+	accessToken
```

Comment:
```suggestion
	roleToken mode = iota
	roleJWT
	accessToken
```
totally NOT important 😅

Reply:
np
I was thinking the same thing and implementing it, but you gave me the answer so it was just right!